frontend/src/metabase/components/ArchivedItem/ArchivedItem.tsx

```diff
@@ -65,7 +72,11 @@ export const ArchivedItem = ({
           </Tooltip>
         )}
         {onDelete && (
-          <Tooltip tooltip={t`Delete this ${type}`}>
+          <Tooltip
+            tooltip={t`Delete this ${getTranslatedEntityName(
+              model,
+            )?.toLowerCase()}`}
```

Comment:
TypeScript complained if I didn't use the optional chain here, because a `model` can be `null`.